tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -15,68 +18,52 @@ runv () {
 }
 
 basearch=$(arch)
-rhelver=$(. /usr/lib/os-release && echo "${RHEL_VERSION}")
-major=$(echo "${rhelver}" | cut -f 1 -d .)
+major=$(. /usr/lib/os-release && echo "${CPE_NAME}" | grep -Eo '[0-9]{1,2}')
 baseurl=
 case "${major}" in
-    8)
-        # TODO: why the heck does centos only support insecure http?!?
-        baseurl=http://mirror.centos.org/centos/8-stream/
-        # TODO avoid hardcoding versions
-        target_kver=4.18.0-383.el8
-        target_kver_rt=4.18.0-394.rt7.179.el8
-        ;;
     9)
         # TODO: why the heck does centos only support insecure http?!?
         baseurl=http://mirror.stream.centos.org/9-stream/
-        # TODO avoid hardcoding versions
-        target_kver=5.14.0-196.el9
-        target_kver_rt=5.14.0-201.rt14.202.el9
+        repo_url=https://raw.githubusercontent.com/openshift/os/bf4383e4929ad9abb67d508a7ffa7cf45e8de5ca/c9s.repo
```

Comment:
Is this specific commit a random one? Should this be https://raw.githubusercontent.com/openshift/os/master/c9s.repo instead?

Reply:
It should, fixed.